src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -5,41 +5,41 @@
 public class FigureSupplier {
     private final int range = 40;
     private ColorSupplier colorSupplier = new ColorSupplier();
-    private final String[] figures = {"square",
-            "rectangle",
-            "righttriangle",
-            "circle",
-            "isoscelestrapezoid"};
+    private final FigureType[] figureTypes = FigureType.values();
     private Random random = new Random();
 
     public Figure getRandomFigure() {
-        int randomFigureIndex = random.nextInt(figures.length);
-        String figure = figures[randomFigureIndex];
+        int randomFigureIndex = random.nextInt(figureTypes.length);
+        String figure = figureTypes[randomFigureIndex].toString();
```

Comment:
Don't use toString() or String.valueOf()(it will call toString() under the hood) for getting the String representation of enum constants. toString() is common for all enum constants. It's better to use the standard method of enum name() that will be returning always String representation of the concrete enum constant.
You also don't need to have a String value here, you can operate on enums: figure will by of type enum and later in if statement you can use enum in comparison, e.g
if (FigureType.SQUARE.equals(figure))